server/prisma/datamodel.prisma

```diff
@@ -0,0 +1,67 @@
+enum TaskStatus {
+  COMPLETED
+  UNCOMPLETED
+}
+
+enum UserRole {
+  NEWBIE
+  BUDDY
+}
+
+type NewbieTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+  notes: String
+}
+
+type BuddyTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+}
+
+type Buddy {
+  id: ID! @id
+  name: String!
+  email: String! @unique
+  role: UserRole! @default(value: BUDDY)
+  password: String!
+  position: String
+  photo: String
+  phoneNumber: String
+  startDate: DateTime
+  notifications: Boolean! @default(value: false)
+  newbiesCount: Int! @default(value: 0)
+  newbies: [Newbie]!
+}
+
+type Newbie {
+  id: ID! @id
+  name: String!
+  email: String! @unique
+  role: UserRole! @default(value: NEWBIE)
+  password: String!
+  position: String
+  photo: String
+  phoneNumber: String
+  startDate: DateTime
+  notifications: Boolean! @default(value: false)
+  buddy: Buddy!
+  notes: String! @default(value: "")
+  tasksRating: Float! @default(value: 0)
+  buddyTasksRating: Float! @default(value: 0)
+  tasksCompletedCount: Int! @default(value: 0)
+  tasksUncompletedCount: Int! @default(value: 0)
```

Comment:
I would replace it with tasksTotal, it seems more useful as from this value you can then obtain other

Reply:
Here's counts of Completed and Uncompleted tasks:
<img width="184" alt="Screenshot 2019-10-04 at 10 52 51" src="https://user-images.githubusercontent.com/5225448/66194665-367a2180-e695-11e9-9072-93f106342e2d.png">
For sure we can add `tasksTotal ` prop as well, just I haven't found any place where we need this to be honest